deployment/Makefile

```diff
@@ -211,8 +211,7 @@ mediasync:
 	@echo "------------------------------------------------------------------"
 	@echo "Syncing media files from live server"
 	@echo "------------------------------------------------------------------"
-	@rsync -av --progress kartoza3:/home/timlinux/production-sites/projecta/django_project/media media
-
+	@rsync -av --progress rischan@5.9.160.107:/home/data/changelog.inasafe.org/deployment/media media
```

Comment:
This change is bad since it will break my ability to sync media. Please revert to the original line and chat to me if it is not working for you.